docs/visualize/tsvb.asciidoc

```diff
@@ -0,0 +1,115 @@
+[[TSVB]]
+== Visualizing your data with TSVB
+
+TSVB is a time series data visualizer that allows you to use the full power of the 
+Elasticsearch aggregation framework. With TSVB, combine an infinite 
+number of aggregations and pipeline aggregations to display complex data in a 
+meaningful way.
+
+NOTE: In Elasticsearch version 7.3.0 and later, the time series data visualizer is now referred to simply as TSVB instead of Time Series Visual Builder.
+
+TSVB comes with these types of visualizations:
+
+Time Series:: A histogram visualization that supports area, line, bar, and steps along with multiple y-axis.
+
+[role="screenshot"]
+image:images/tsvb-screenshot.png["Time series visualization"]
+
+Metric:: Displays the latest number in a data series.
```

Comment:
Make this consistent with the other descriptions?  A metric that displays the latest number in a data series. This is also true of Markdown and Table.